ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/ProtocolSpec.java

```diff
@@ -237,6 +241,15 @@ public Wei getBlockReward() {
     return blockReward;
   }
 
+  /**
+   * EIP 158 prohibits adding new "empty" accounts to the account trie.
+   *
+   * @return if EIP158 is in effect for this spec.
+   */
+  public boolean isEIP158() {
```

Comment:
Naming quibbles - maybe this could be `isEIP158Enabled`.  Not sure if it would be a good idea name it something more "meaningful" like `prohibitEmptyAccounts`?

Reply:
I'll go with `IsEIP158Enabled` when I fix the acceptance tests.  My long term vision is that we have an `isEIPEnabled(int eip)` option and we track all the EIPs.  That way we can speculatively include EIPs in the code base and use the EIP registry as feature flags.  But since we care about only one at this point I'll put the number in the name,

Reply:
Turns out our clique tests dependency, so labeling it as EIP158 treatment is inaccurave, so I went with skipZeroBlockRewards.